src/config/routes/index.js

```diff
@@ -1,17 +1,14 @@
-const jwt = require('koa-jwt');
-
-const config = require('config');
-
-const { urlToken, user } = require('./middlewares');
+const { auth } = require('./middlewares/user.middleware');
```

Comment:
Only keep the logic to attach user to ctx.state.user (rather null if token is invalid or missing or user object)

Move auth logic to separate middleware (middlewares/auth.middleware)
Throw 401 only if ctx.state.user is not present